content/PIV/Guides/Securing_SSH_with_OpenPGP_or_PIV.adoc

```diff
@@ -1,22 +1,10 @@
-residing on a physical YubiKey. With a YubiKey, the user’s SSH key data can be secured using OpenPGP and/or PIV.
-Typically, deployment considerations or the particular use case will dictate which application to use.
+OpenSSH is one of the most widely used utilities suites based on the SSH protocol, which provides a secure channel
+over an unsecured network in a client-server architecture. Securing communication with remote systems via SSH can be done by using key-based authentication with the user’s key residing on a physical YubiKey. The user’s SSH key data on the YubiKey can be secured using OpenPGP and/or PIV. Typically, deployment considerations or the particular use case will dictate which application to use.
 
-If you use the OpenPGP option for SSH, the same key will be used for both authentication and signing. This is
-demonstrated by Git, which uses GnuPG, that uses the same key to grant access to a repository via SSH and to sign a
-commit. However, the OpenPGP option is more complicated to set up properly since the SSH communication is done through
-the GnuPG agent. The PIV option, on the other hand, would probably be easier for anyone who does not have a reason to
-use the same key for authentication and signing. Furthermore, for the PIV option, you can use the opensc-pkcs11 module,
-or our own ykcs11 module. Our ykcs11 module is a lighter software package, so it would probably offer an easier path
-for someone who does not already have the OpenSC software installed. The same goes for OpenPGP vs PIV as well, if you
-already have GnuPG working.
+If you use the *OpenPGP* option for SSH, the same key will be used for both authentication and signing. Git, which uses GnuPG, exemplifies this approach in that it uses a single public key both to authenticate an SSH connection used to access the repository--for example, when pushing commits--and for verifying the signatures on those commits. Because the SSH communication is done through the GnuPG agent, the OpenPGP option is more complicated than PIV to set up. However, developers who use Git to sign their commits typically choose the OpenPGP path because they already have it up and running.
 
-
-For a developer who uses Git to sign his/her commits, the OpenPGP path is typically the natural one, because GnuPG
-needs to be working to enable the use of Git signatures. With OpenPGP, it is only necessary to upload a single public
-key, which Git then uses both to authenticate an SSH connection (used when accessing the specific repository, for
-example when pushing one’s commits) and for verifying the signatures on Git commits.
+The *PIV* option is easier for those who have no reason to use the same key for both authentication and signing. The PIV option also offers the opensc-pkcs11 module, or Yubico's own ykcs11 module, which is a lighter software package than the the OpenSC software.

```

Comment:
so we don't need "#" in the module, although we do need it in the interface
opensc-pkcs11/ykcs11 module             ---------                 PKCS#11 interface

Reply:
Correct. That's just because this is how they are named:
PKCS#11 is part of a suit of standards separated by number. They span from PKCS#1 through PKCS#15. For example, PKCS#12 describes a file format to store private keys, and PKCS#7 describes message syntax to sign or decrypt a message...etc 
The opensc-pkcs11 and ykcs11 are software that implement the PKCS#11 standard. We happen to call ours "ykcs11"